test/unit/tasks/download-assets.test.js

```diff
@@ -10,13 +10,75 @@ const tmpDirectory = resolve(tmpdir(), 'contentful-import-test')
 
 nock('https://images.contentful.com')
   .get('/kq9lln4hyr8s/2MTd2wBirYikEYkIIc0YSw/7aa4c06f3054996e45bb3f13964cb254/rocka-nutrition.png')
-  .times(2)
+  .times(6)
   .reply(200)
 
 nock('https://images.contentful.com')
   .get('/does-not-exist.png')
   .reply(404)
 
+function getAssets ({ existing = 0, nonExisting = 0, missingUrl = 0 } = {}) {
+  const existingUrl = '//images.contentful.com/kq9lln4hyr8s/2MTd2wBirYikEYkIIc0YSw/7aa4c06f3054996e45bb3f13964cb254/rocka-nutrition.png'
```

Comment:
we could probably store these strings as global `const`s in this file and reuse them in the nock calls above (line 12 and 16)